crates/polars-plan/src/dsl/function_expr/boolean.rs

```diff
@@ -81,27 +81,27 @@ impl From<BooleanFunction> for SpecialEq<Arc<dyn SeriesUdf>> {
     fn from(func: BooleanFunction) -> Self {
         use BooleanFunction::*;
         match func {
-            Any { ignore_nulls } => map!(any, ignore_nulls),
             All { ignore_nulls } => map!(all, ignore_nulls),
-            IsNot => map!(is_not),
-            IsNull => map!(is_null),
-            IsNotNull => map!(is_not_null),
+            AllHorizontal => wrap!(all_horizontal),
```

Comment:
Please refrain from including random refactors like these in your PRs. While you might prefer alphabetical order, the functions were already ordered by function (e.g. IsFinite next to IsInfinite). Now we have to debate the merits of this change, while I really just want to review the deprecation...

Reply:
Sorry about that. I'll revert those changes.